From 11728cde40187219fcabddda25aac9f437ce2da5 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:17 -0400
Subject: [tools] perf: Remove PR_TASK_PERF_EVENTS_* from perf.h

Message-id: <1410181752-25631-193-git-send-email-jolsa@redhat.com>
Patchwork-id: 91803
O-Subject: [PATCH RHEL7.1 BZ1134356 192/307] perf tools: Remove PR_TASK_PERF_EVENTS_* from perf.h
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 36446f4bfe613ad1da00f06081d410d62c07d334
Author: Jiri Olsa <jolsa@kernel.org>
Date: Mon May 5 12:37:05 2014 +0200

description
===========
It's defined in include/uapi/linux/prctl.h header.
Also it was never used in perf tool.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index 622949d..1bf9537 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -176,13 +176,6 @@
 #include <linux/types.h>
 #include <linux/perf_event.h>
 
-/*
- * prctl(PR_TASK_PERF_EVENTS_DISABLE) will (cheaply) disable all
- * counters in the current task.
- */
-#define PR_TASK_PERF_EVENTS_DISABLE   31
-#define PR_TASK_PERF_EVENTS_ENABLE    32
-
 #ifndef NSEC_PER_SEC
 # define NSEC_PER_SEC   1000000000ULL
 #endif
-- 
1.7.1